docs/src/languages/php/extensions.md

```diff
@@ -60,110 +61,111 @@ You can disable those by adding them to the `disabled_extensions` list.
+| gearman          | *   | *   | *   |     |     |     |     |     |     |
+| geoip            | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| gettext          |     |     |     | *   | *   | *   | *   | *   | *   |
+| gmp              | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| http             | *   | *   |     |     |     |     |     | *   |     |
+| iconv            |     |     |     | *   | *   | *   | *   | *   | *   |
+| igbinary         |     |     |     | *   | *   | *   | *   | *   | *   |
+| imagick          | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| imap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| interbase        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| intl             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| ioncube          |     |     |     | *   | *   | *   |     |     |     |
+| json             |     |     | *   | *   | *   | *   | *   | *   | *   |
+| ldap             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| mailparse        |     |     |     | *   | *   | *   |     | *   |     |
+| mbstring         |     |     |     | *   | *   | *   | *   | *   | *   |
+| mcrypt           | *   | *   | *   | *   | *   |     |     |     |     |
+| memcache         | *   | *   | *   |     |     |     |     |     |     |
+| memcached        | *   | *   | *   | *   | *   | *   |     | *   |     |
+| mongo            | *   | *   | *   |     |     |     |     |     |     |
+| mongodb          |     |     |     | *   | *   | *   | *   | *   |     |
+| msgpack          |     |     | *   | *   | *   | *   |     | *   |     |
+| mssql            | *   | *   | *   |     |     |     |     |     |     |
+| mysql            | *   | *   | *   |     |     |     |     |     |     |
+| mysqli           | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| mysqlnd          | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| newrelic         |     |     | *   | *   | *   | *   | *   | *   | *   |
+| oauth            |     |     |     | *   | *   | *   | *   | *   |     |
+| odbc             | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| opcache          |     | *   | *   | *   | *   | *   | *   | *   | *   |
+| openssl          |     |     |     |     |     |     |     |     |     |
+| pdo              | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_dblib        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_firebird     | *   | *   | *   | *   | *   | *   | *   | *   |     |
+| pdo_mysql        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_odbc         | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_pgsql        | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_sqlite       | *   | *   | *   | *   | *   | *   | *   | *   | *   |
+| pdo_sqlsrv       |     |     |     | *   | *   | *   |     | *   |     |
```

Comment:
Is this still pending?  The gap on 7.3 is... weird.

Reply:
Hum you are right we have the extension for 7.3 fixed in babdeb75 and yep the php8 release is still beta